oidc_auth/authentication.py

```diff
@@ -68,7 +68,11 @@ def get_bearer_token(self, request):
 
     @cache(ttl=api_settings.OIDC_BEARER_TOKEN_EXPIRATION_TIME)
     def get_userinfo(self, token):
-        response = requests.get(self.oidc_config['userinfo_endpoint'],
+        userinfo_endpoint = self.oidc_config.get('userinfo_endpoint', api_settings.USERINFO_ENDPOINT)
+        if userinfo_endpoint in ['', None, [], (), {}]:
```

Comment:
I would rather this be `if not userinfo_endpoint:`, which is much more readable and probably what you want anyway.